docs/quickstart.rst

```diff
@@ -124,7 +124,7 @@ After downloading the dataset and unzipping it in your current working directory
 
 The machine learning workflow is the art of creating *model selection triples*, a combination of features, algorithm, and hyperparameters that uniquely identifies a model fitted on a specific data set. As part of our feature selection, we want to identify features that have a linear relationship with each other, potentially introducing covariance into our model and breaking OLS (guiding us toward removing features or using regularization). We can use the Rank2D_ visualizer to compute Pearson correlations between all pairs of features as follows:
```

Comment:
To reference the API documentation for RankD visualizers you could replace `Rank2D_` with `` :doc:`Rank2D <api/features/rankd>` `` (note the backticks in the example) -- this allows us to use the `sphinx.ext.intersphinx` extension to reference other documents and you can remove the reference definition on line 127. 

The problem with this is that this will go to the top of the RankD documentation page, and the section on Rank2d is farther down. In order to go to that specific section using vertical references you'd have to put a named anchor in that document and use `` :ref:`anchor-name` ``.

Reply:
Can you please check if i did it correct?

Reply:
Apologies @Naba7 if we're missing something, but it doesn't appear that you've pushed the commits with the corrections that @bbengfort requested. Is it possible you pushed the changes to a different branch? If so, can I recommend that we close this PR and have you open a new PR from your documentation bugfix branch into our develop branch (this is our convention for contributors).